hello-thirdparty/app/build.gradle

```diff
@@ -1,9 +1,20 @@
 apply plugin: 'com.android.model.application'
 
-
 def gpg_sdk_path = file(project(':thirdparty:gpg-sdk').projectDir).absolutePath + "/gpg-cpp-sdk/android"
 
 model {
+
+    repositories {
+        libs(PrebuiltLibraries) {
+            gpg {
+                headers.srcDir "${gpg_sdk_path}/include"
+                binaries.withType(SharedLibraryBinary) { // NOTE: this block should be "StaticLibraryBinary"/staticLibraryFile - but SharedLibraryBinary works and StaticLibraryBinary doesn't as of 0.6.0-alpha1
```

Comment:
@rschiu any idea why?

What was the behavior w/ 0.4? (maybe the feature wasn't available yet?).